app/services/email_service/api/addresses.rb

```diff
@@ -35,15 +37,38 @@ def get_user_defined(community_id:)
     end
 
     def create(community_id:, address:)
-      Result::Success.new(
-        with_formats(
-          AddressStore.create(
-          community_id: community_id,
-          address: {verification_status: :none}.merge(address))))
+      create_in_status = @ses_client ? :none : :verified
```

Comment:
`get_sender` needs to be changed so that it allows status `:none` if ses client is `nil`